README.md

```diff
@@ -18,6 +18,8 @@
 
 </p>
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/pipx.svg)](https://repology.org/metapackage/pipx/versions)
```

Comment:
IMO this would look nicer if was split into 3 columns. I'd also exclude unsupported, to only keep relevant entries.

```suggestion
[![Packaging status](https://repology.org/badge/vertical-allrepos/pipx.svg?columns=3&exclude_unsupported=1)](https://repology.org/metapackage/pipx/versions)
```